src/stream/src/lib.rs

```diff
@@ -54,3 +54,15 @@ pub mod task;
 
 #[cfg(test)]
 risingwave_expr_impl::enable!();
+
+mod consistency {
+    use std::sync::LazyLock;
+
+    use risingwave_common::util::env_var::env_var_is_true;
+
+    static INSANE_MODE: LazyLock<bool> = LazyLock::new(|| env_var_is_true("RW_ENABLE_INSANE_MODE"));
```

Comment:
Use env var instead of compiling cfg is to make future longevity test in insane mode easy to setup.

Reply:
Similar to https://github.com/risingwavelabs/risingwave/pull/15796#discussion_r1533258768, maybe better called `RW_UNSAFE_ENABLE_INSANE_MODE`